Description: set correct path to perl interpreter in examples
 The supplied expample scipts assume that perl is installed
 in /usr/local/bin which is not the case in Debian.
 This patch changes the path to /usr/bin/perl in all supplied
 example scripts and corrects a comment in scripts/identd.perl5.
Author: Raoul Gunnar Borenius <borenius@dfn.de>
Forwarded: not-needed
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Last-Update: 2012-04-17
--- a/scripts/00README
+++ b/scripts/00README
@@ -33,9 +33,9 @@
 	Path to:	Is:
 	=======		==
 
-	Perl 4		/usr/local/bin/perl4
+	Perl 4		/usr/bin/perl
 
-	Perl 5		/usr/local/bin/perl
+	Perl 5		/usr/bin/perl
 
 If your system doesn't support the command interpreter feature,
 you'll have to supply the scripts to your Perl interpreter on its
--- a/scripts/big_brother.perl5
+++ b/scripts/big_brother.perl5
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #+##############################################################################
 #                                                                              #
 # File: big_brother.perl                                                       #
--- a/scripts/count_pf.perl
+++ b/scripts/count_pf.perl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # count_pf.perl-- run lsof in repeat mode and count processes and
 #		  files
--- a/scripts/count_pf.perl5
+++ b/scripts/count_pf.perl5
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # count_pf.perl5 -- run lsof in repeat mode and count processes and
 #		    files
--- a/scripts/identd.perl5
+++ b/scripts/identd.perl5
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 ###################################################################
 # identd.perl5 : An implementation of RFC 1413 Ident Server
 #                using Vic Abell's lsof.
@@ -11,7 +11,7 @@
 # - Multiple instances of the server are not a performance penalty
 #   since they shall use lsof's cacheing mechanism. (compare with
 #   Peter Eriksson's pidentd)
-# - assumes 'lsof' binary in /usr/local/sbin
+# - assumes 'lsof' binary in $PATH
 # - Command line arguments :
 #   -t TIMEOUT Number of seconds to wait for a query before aborting.
 #              Default is 120.
--- a/scripts/idrlogin.perl
+++ b/scripts/idrlogin.perl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Id: idrlogin.perl,v 1.5 2001/11/18 12:20:46 abe Exp $
 #
--- a/scripts/idrlogin.perl5
+++ b/scripts/idrlogin.perl5
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Id: idrlogin.perl5,v 1.5 2001/11/18 12:20:46 abe Exp $
 #
--- a/scripts/list_NULf.perl5
+++ b/scripts/list_NULf.perl5
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Id: list_NULf.perl5,v 1.5 2000/07/14 17:03:37 abe Exp $
 #
--- a/scripts/list_fields.perl
+++ b/scripts/list_fields.perl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl4
+#!/usr/bin/perl
 #
 # $Id: list_fields.perl,v 1.5 2000/07/14 17:03:37 abe Exp $
 #
--- a/scripts/shared.perl5
+++ b/scripts/shared.perl5
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Id: shared.perl5,v 1.4 2001/11/18 12:20:46 abe Exp $
 #
--- a/scripts/watch_a_file.perl
+++ b/scripts/watch_a_file.perl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # watch_a_file.perl -- use lsof -F output to watch a specific file
 #		       (or file system)
